libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.html

```diff
@@ -3,21 +3,30 @@
   aria-haspopup="listbox"
   class="sky-autocomplete"
   role="combobox"
-  [attr.aria-expanded]="isDropdownVisible | async"
+  [attr.aria-expanded]="
+    isOpen && (showActionsArea || (isResultsVisible | async))
+  "
   [attr.aria-labelledby]="ariaLabelledBy"
-  [attr.aria-controls]="(isDropdownVisible | async) ? resultsListId : null"
+  [attr.aria-controls]="
+    isOpen && (showActionsArea || (isResultsVisible | async))
+      ? resultsListId
+      : null
+  "
   [attr.id]="resultsWrapperId"
 >
   <ng-content></ng-content>
 </div>
 
 <ng-template #resultsTemplateRef>
+  <!-- We do not want the blur to fire here so that the dropdown does not close. This is why we are preventing the mousedown default. -->
   <div
     class="sky-autocomplete-results-container"
+    [attr.id]="resultsListId"
```

Comment:
This was moved due to the `aria-controls` needing to reference the whole dropdown and not just the results area.